package.json

```diff
@@ -4,10 +4,11 @@
   "description": "mervin's portion of etsy mock",
   "main": "index.js",
   "scripts": {
-    "test": "jest --detectOpenHandles",
```

Comment:
what made you change out open handles for your new test script?  Does that new setup still work for async tests

Reply:
or did you get rid of that because you used supertest?

Reply:
Jest wont close and hangs up since there's something in the background that keeps running. So I added forceExit